app/src/main/java/io/homeassistant/companion/android/sensors/SensorsSettingsFragment.kt

```diff
@@ -233,7 +233,8 @@ class SensorsSettingsFragment : PreferenceFragmentCompat() {
 
         if (showOnlyEnabledSensors) {
             val checkable = menu.findItem(R.id.action_show_only_enabled_sensors)
-            checkable.isChecked = true
+            if (checkable != null)
+                checkable.isChecked = true
```

Comment:
Can't this just be this:
```suggestion
            checkable?.isChecked = true
```